docs/podman-image-trust.1.md

```diff
@@ -1,7 +1,7 @@
 % podman-image-trust "1"
 
 # NAME
-podman\-trust - Manage container registry image trust policy
+podman\-trust - Manage container image trust policy
```

Comment:
Why are you removing registry.  podman trust is identifying whether you trust registry images, ones on your current system are not checked?

Reply:
I'll add it back in.  I'd thought it was redundant, but had thought local vs registry images.